sdks/python/apache_beam/coders/standard_coders_test.py

```diff
@@ -74,7 +77,7 @@ class StandardCodersTest(unittest.TestCase):
           lambda x: IntervalWindow(
               start=Timestamp(micros=(x['end'] - x['span']) * 1000),
               end=Timestamp(micros=x['end'] * 1000)),
-      'urn:beam:coders:stream:0.1': lambda x, parser: map(parser, x),
+      'urn:beam:coders:stream:0.1': lambda x, parser: list(map(parser, x)),
```

Comment:
<!--new_thread; commit:5fe62966c143f8060ff75ab7edb2258f1e0df55e; resolved:0-->
Do we need this `list` call here?  This may not be intended behavior, to materialize the list, since the stream may be a long sequence that doesn't fit in memory.

Reply:
I don't think we need it. Seems like I missed this one. Thanks for pointing out.

Reply:
There's one test that fails without the list call. I'll look into it.